modules/110-istio/templates/kiali/service.yaml

```diff
@@ -10,8 +10,10 @@ spec:
   - name: http
     protocol: TCP
     port: 20001
+    targetPort: api-port
```

Comment:
change to "api" both in service and deployment pls.

Reply:
Done!